### PR TITLE
Update data node management with example of moving a chunk

### PR DESCRIPTION
**Pull request by erimatnor**
The data node management section now has a basic description of moving
a chunk between data nodes. This complements the section on adding new
data nodes (elasticity) with information on how to actually get data
to a new node.

The information on copying and moving chunks is now split across the
data node management section and the native replication section. In
the future, we might want to rewrite this to have everything about
copying and moving chunks in a single place.

# Version

Which documentation version does this PR apply to?

- [x ] Latest (Default)
- [x ] Version 2.4

### CODE REVIEW

File: timescaledb/how-to-guides/distributed-hypertables/manage-data-nodes.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ WHERE hypertable_name = 'conditions';
  conditions      | {data_node_1,data_node_2,data_node_3}
 ```
 
-If additional data nodes are added to a distributed database, the data
-nodes are not automatically associated with existing distributed
+If additional data nodes are added to a database, the data nodes are
+not automatically associated with existing distributed
 hypertables. Instead, you need to explicitly *attach* a data node
 using [`attach_data_node`][attach_data_node]:
```
Comment (Loquacity) [inline timescaledb/how-to-guides/distributed-hypertables/manage-data-nodes.md:25]:
```suggestion
If you add additional data nodes to a database, they are
not automatically associated with existing distributed
hypertables. Instead, you need to explicitly attach a data node
using [`attach_data_node`][attach_data_node]:
```

File: timescaledb/how-to-guides/distributed-hypertables/manage-data-nodes.md [hunk 2]
```diff
@@ -31,10 +31,10 @@ SELECT attach_data_node('node3', hypertable => 'conditions');
 
 When attaching a data node, the partitioning configuration of the
```
Comment (Loquacity) [inline timescaledb/how-to-guides/distributed-hypertables/manage-data-nodes.md:32]:
```suggestion
When you attach a data node, the partitioning configuration of the
```

File: timescaledb/how-to-guides/distributed-hypertables/manage-data-nodes.md [hunk 3]
```diff
@@ -31,10 +31,10 @@ SELECT attach_data_node('node3', hypertable => 'conditions');
...
 distributed hypertable is also updated to account for the additional
-data node (i.e., the number of space partitions is automatically increased to
-match), unless the function parameter `repartition` is set to
-`FALSE`. The updated partitioning configuration ensures that the data
-node will be able to take on newly created chunks.
+data node (i.e., the number of space partitions is automatically
+increased to match), unless the function parameter `repartition` is
+set to `FALSE`. The updated partitioning configuration ensures that
+the data node will be able to take on newly created chunks.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/distributed-hypertables/manage-data-nodes.md:37]:
```suggestion
data node (the number of space partitions are automatically
increased to match), unless the function parameter `repartition` is
set to `FALSE`. The updated partitioning configuration ensures that
the data node is able to take on newly created chunks.
```

File: timescaledb/how-to-guides/distributed-hypertables/manage-data-nodes.md [hunk 4]
```diff
@@ -48,7 +48,32 @@ hypertable. To be able to detach a data node, ensure that either (1)
 all its data is deleted first, or (2) the data is replicated on other
 data nodes (see the next section on native replication).
 
+## Moving data between nodes
 
+When attaching a new data node to a distributed hypertable, it might
+desirable to move data to the new node in order to free up storage on
+existing nodes and make queries use the added capacity. Data can be
+moved between nodes at the chunk level:
```
Comment (Loquacity) [inline timescaledb/how-to-guides/distributed-hypertables/manage-data-nodes.md:56]:
```suggestion
When you attach a new data node to a distributed hypertable, you might
prefer to move data to the new node to free up storage on the
existing nodes and make queries use the added capacity. Data can be
moved between nodes at the chunk level:
```

File: timescaledb/how-to-guides/distributed-hypertables/manage-data-nodes.md [hunk 5]
```diff
@@ -48,7 +48,32 @@ hypertable. To be able to detach a data node, ensure that either (1)
...
+```sql
+CALL timescaledb_experimental.move_chunk('_timescaledb_internal._dist_hyper_1_1_chunk', 'data_node_3', 'data_node_2');
+
+```
+
+<highlight type="warning"> 
+The ability to move chunks between data nodes is an experimental
+feature that is under active development. Use in production
+environments at your own risk.
+</highlight>
```
Comment (Loquacity) [inline timescaledb/how-to-guides/distributed-hypertables/manage-data-nodes.md:67]:
```suggestion
<highlight type="warning"> 
The ability to move chunks between data nodes is an experimental
feature that is under active development. We recommend that you
do not use this feature in a production environment.
</highlight>
```

**Review by Loquacity — CHANGES_REQUESTED** [review]
Just small edits, non-blocking.

**Review by NunoFilipeSantos — APPROVED** [review]
I agree with @Loquacity 's suggestions.

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
@erimatnor Ready to merge 👍

**Comment by NunoFilipeSantos** [comment]
To follow the release of 2.4.

**Pull request merged**